elliott/elliottlib/cli/find_builds_cli.py

```diff
@@ -707,3 +720,87 @@ async def find_builds_konflux(runtime, payload):
     if len(records) != len(image_metas):
         raise ElliottFatalError(f"Failed to find Konflux builds for {len(image_metas) - len(records)} images")
     return records
+
+
+async def find_builds_konflux_all_types(runtime):
```

Comment:
please return a dict with keys, instead of tuples in arbitrary order.
And add return type annotation